cmd/broker/ingress/main.go

```diff
@@ -156,3 +188,23 @@ func (r *runnableServer) Start(<-chan struct{}) error {
 	r.logger.Info("Ingress Listening...", zap.String("Address", r.s.Addr))
 	return r.s.ListenAndServe()
 }
+
+// TODO this should be removed once we update the interfaces and start using cloudevents.Event instead of Message.
+func cloudEventFrom(m *provisioners.Message) cloudevents.Event {
```

Comment:
oh man you should use the cloudevents lib: https://github.com/cloudevents/sdk-go

Reply:
Yes, I put that in the TODO. 
BTW how do I convert the provisioners.Message to a cloudevents.Event if I don't inspect the headers? 
Adam was working on changing the API to not receive a Message as an arg, but a cloudevent... I was waiting for that change